task.go

```diff
@@ -141,7 +141,7 @@ func (e *Executor) RunTask(ctx context.Context, call taskfile.Call) error {
 			}
 
 			if upToDate && preCondMet {
-				if !e.Silent {
+				if e.Verbose || (!t.Silent && !e.Taskfile.Silent && !e.Silent) {
```

Comment:
I see this was already being respected for the `--silent`, but not for the Taskfile or task level attribute.

One more reason to proceed in name of consistency. 🙁